locale/en/site.json

```
@@ -82,7 +82,11 @@
       "verify-text": "How to verify"
     },
     "install-on-linux": {
-      "text": "Installing Node.js via binary archive"
+      "text": "Installing Node.js via binary archive on Linux"
```

```suggestion
      "text": "Installing Node.js via binary archive"
```

I don't think there is a need to make this copy change (is it perhaps left over from the earlier version?

---

Yes - looks like a leftover. :)